nameresolution/metadata.go

```diff
@@ -16,6 +16,8 @@ package nameresolution
 import "github.com/dapr/components-contrib/metadata"
 
 const (
+	// TODO: REMOVE THESE AFTER RUNTIME IS CHANGED
```

Comment:
Reviewers: The MDNS constants below are unused but I'm not removing them in this PR because they have to be changed in the runtime first by pinning the latest components-contrib. Otherwise the runtime will break and so will our certification tests